README.md

```diff
@@ -15,6 +15,12 @@ Install affjax-node with [Spago](https://github.com/purescript/spago):
 spago install affjax-node
 ```
 
+And also install `xhr2` (see [#5](https://github.com/purescript-contrib/purescript-affjax-node/issues/5#issuecomment-1272142145))
+```sh
+npm init
+npm i xhr2
+```
+
```

Comment:
Instead of linking to the thread, can this just be updated to say that the NPM dependency `xhr2` must be installed since it's one of the dependencies of the library? And that users can install it via `npm i xhr2` or `yarn add xhr2`?

Can the `npm init` be dropped?

Reply:
Wow, didn't know `npm init` can be omitted